src/components/Display.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+const Display = (props) => {
+  const { value } = props;
+  return <div>{value}</div>;
+};
+
```

Comment:
- Enforce the `propType` of `"value"` to be string   ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png))
- Also enforce the default value to be ZERO ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png))

You can research about these 2 small concepts (propTypes, defaultProps)